MediaGalleryUi/view/adminhtml/web/template/image/image-details.html

```diff
@@ -16,6 +16,10 @@ <h3 class="image-title" data-bind="text: image().title"></h3>
                 <span class="type" data-ui-id="content-type" data-bind="text: image().content_type"></span>
             </div>
         </div>
+        <div class="filename image-details-section">
+            <h3 data-bind="i18n: 'Filename'"></h3>
```

Comment:
Can you please add filename verification the corresponding MFTF action group

Reply:
@sivaschenko ... I've added the MFTF test